### ISSUE TITLE
[BUG] Card description icon is missing from the card in the app

### ISSUE THREAD

**Issue by user8446** [stefan-niedermann/nextcloud-deck#1583]
### This issue respects the following points:

- [X] I have read the [FAQ](https://github.com/stefan-niedermann/nextcloud-deck/blob/master/FAQ.md)
- [X] I have searched for existing issues
- [X] The issue affects the Deck Android app, neither the [Deck server app](https://github.com/nextcloud/deck/issues/) nor the [Nextcloud Android app](https://github.com/nextcloud/android/issues).

### Describe the bug

If a card has a description, there is a description icon on the card to indicate this. This is shown in the  browser, but not in the app.

However, if there is also an attachment on the card both the description icon and the attachment icon will then both show. 

**Browser view:**

![Screenshot 2024-02-14 104647](https://github.com/stefan-niedermann/nextcloud-deck/assets/18019245/0c88f226-012e-458e-87c5-1fe277f02639)

**Mobile app view:**

![Screenshot_20240214-104558](https://github.com/stefan-niedermann/nextcloud-deck/assets/18019245/fa3d35a3-0e3c-4e36-9746-dd5210586b4c)

### Expected behavior

Description icon appears in the mobile app like with the browser view.

### Deck Android version

1.23.4

### Deck server version

1.11.3

### Nextcloud Android version

3.27.0

### Nextcloud version

27.1.6

### Device

Pixel 6a

### Android Version

14

### App Store

- [ ] Google Play Store
- [ ] Google Play Store (Beta channel)
- [X] F-Droid
- [ ] Huawei App Gallery

**Comment by stefan-niedermann** [comment]
Yeah, this has been implemented different to the Deck server app by intention. The idea was to use one-line-cards if possible to save some vertical space.

Compared to attachments, comments and tasks the pure presence of any description content seemed to be not important enough to justify the additional vertical space.

However, consistency with the Deck server app is a goal for us, so I am ready to rethink the decision. Especially given the Deck server app display the due date / done date *always* in a separate line, we should think about doing the same.

**Comment by user8446** [comment]
Thank you, I agree with the consistency. There is also the compact mode toggle in the app settings so the user can still switch to the smaller cards if desired.
